include/pinocchio/algorithm/frames.hpp

```diff
@@ -125,6 +125,11 @@ namespace pinocchio
    *             This is different from the "spatial" acceleration in that centrifugal effects are accounted for.
    *             You must first call pinocchio::forwardKinematics to update placement, velocity and acceleration values in data structure.
    *
+   * @remark     In the context of a frame placement constraint \f$J(q) a + \dot{J}(q, v) v = 0\f$,
+   *             the classical acceleration of the frame corresponds to the \f$\dot{J}(q, v) v\f$ term.
+   *             It is significantly more efficient to call this function than,
+   *             say, to apply the matrix \f$\dot{J}(q, v)\f$ (from getFrameJacobianTimeVariation) to the velocity \f$v\f$.
```

Comment:
I am not sure about I understand correctly this remark. What do you mean by "corresponds" that the output of this function is equal to $\dot{J}(q, v) v$ ?

Reply:
Thank you, yes I meant equals (simpler and more to the point :+1:):

```suggestion
   * @remark     In the context of a frame placement constraint \f$J(q) a + \dot{J}(q, v) v = 0\f$,
   *             the classical acceleration of the frame is equal to \f$\dot{J}(q, v) v\f$.
   *             It is then significantly more efficient to call this function than,
   *             say, to apply the matrix \f$\dot{J}(q, v)\f$ (from getFrameJacobianTimeVariation) to the velocity \f$v\f$.
```

This is something discussed in https://github.com/stack-of-tasks/pinocchio/issues/1395. It is also in the code of [TaskSE3Equality::compute](https://github.com/stack-of-tasks/tsid/blob/0a8357568085d43f5a7c518d8cb60958406d0484/src/tasks/task-se3-equality.cpp#L155) in TSID.

Reply:
What is discussed in https://github.com/stack-of-tasks/pinocchio/issues/1395#issuecomment-779978792 is that the two are equal **only** if a zero acceleration is passed to `forwardKinematics`, see the code in https://github.com/stack-of-tasks/pinocchio/issues/1395#issuecomment-779978792 .  I am not super-familiar with the TSID codebase, but I guess something similar is happening there.

In general, it does not match my experience (documented in https://github.com/ami-iit/idynfor/blob/5c97708360a4f4e3338bba4203893dcea7295c5c/doc/theory_background.md#frame-acceleration) with Pinocchio that `getFrameClassicalAcceleration` returns just that bias term. From what I understand both the classical and the spatial acceleration have a $J \dot{v}$ and $\dot{J} \nu$, simply the definition of the jacobians between the two are different.

Reply:
And both `getFrameClassicalAcceleration` and `getFrameAcceleration` are equal to $\dot{J}\nu$, using the corresponding definition of $J$ and if $\dot{\nu}$ is set to zero.